src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -91,6 +91,10 @@ public String userRoleClassification() {
         return caseDefinitionHost + "/api/user-role?role={userRole}";
     }
 
+    public String userRolesClassificationsURL() {
+        return caseDefinitionHost + "/api/user-roles/{roles}";
```

Comment:
I would like to open a new story to rename this endpoint in definition store. Which I find confusing. 
This really should be 
`/api/security-classifications/{roles}`

What do you think Fatih?

Reply:
https://tools.hmcts.net/jira/browse/RDM-2319

Reply:
that looks better, thanks